tracker/tracker_debugger_api.yaml

```diff
@@ -1318,3 +1374,912 @@ components:
+          description: 'Cookie ID sent in the _id parameter'
+          type: string
+          example: eddf6b9aa9d8fb32
+        user_id:
+          description: 'User ID sent in the uid parameter'
+          type: string
+          example: 'particular-user_login'
+        body:
+          description: 'Request body payload'
+          type: string
+          example: "idsite=1&rec=0&action_name=This%20won%27t%20be%20recorded"
+        debug:
+          description: 'States whether request is used for debugging'
+          type: boolean
+          default: false
+          example: true
+    TrackingParameters:
+      description: A tracking log
+      required:
+        - idsite
+      type: object
+      properties:
+        idsite:
+          description: ID of the tracked app/website
+          type: string
+          example: 4
+        _idts:
+          description: 'Time of first visit'
+          type: integer
+          minimum: 0
+          maximum: 4294967295
+          example: 1577836800
+        _viewts:
+          description: 'Time of previous visit'
+          type: integer
+          minimum: 0
+          maximum: 4294967295
+          example: 1580342400
+        _idvc:
+          description: 'Visit counter'
+          type: integer
+          minimum: 0
+          maximum: 4294967295
+          example: 2
+        url:
+          description: 'URL of the action'
+          type: string
+          example: 'https://piwik.pro/'
+        urlref:
+          description: 'HTTP referrer'
+          type: string
+          example: 'https://clearcode.cc/'
+        res:
+          description: 'Resolution of the visitor''s device in pixels'
+          type: string
+          example: '1920x1080'
+        idgoal:
+          description: 'Goal ID. Signifies that a goal has been reached and tracks its conversion.'
+          type: string
+          example: '12'
+        revenue:
+          description: Revenue value of achieved goal
+          type: number
+          minimum: -1.0e+28
+          maximum: 1.0e+28
+          example: 7.12
+        ping:
+          description: 'Enables a special heartbeat mode for the tracking request'
+          type: integer
+          minimum: 0
+          maximum: 255
+          example: 1
+        send_image:
+          description: 'Request response content'
+          type: boolean
+          example: true
+        _cvar:
+          description: '[Custom variables](https://help.piwik.pro/analytics/custom-variables/) set on the ``visit`` scope.'
+          type: string
+          example: '{"1":["AspectRatio","16:9"],"2":["Height","1080"],"3":["Width","1920"]}'
+        cvar:
+          description: '[Custom variables](https://help.piwik.pro/analytics/custom-variables/) set on the ``page`` scope.'
+          type: string
+          example: '{"1":["AspectRatio","16:9"],"2":["Height","1080"],"3":["Width","1920"]}'
+        bots:
+          description: 'Bots tracking enabled / disabled'
```

Comment:
This one is no longer used.

Reply:
Liar, liar...
Both [CF](https://github.com/PiwikPRO/PPMS-Trucker-CollectorFront/blob/master/src/tracker/query_params.rs#L42) and [CSA](https://github.com/PiwikPRO/PPMS-Trucker-CollectorStorageApi/blob/master/src/log_repository/logs.rs#L232) still support it, so it's possible it can pop up in the endpoint output thus should be documented.